static/js/apps/eval_rig/app.tsx

```diff
@@ -102,17 +101,17 @@ export function App(props: AppPropType): JSX.Element {
       );
     }
     Promise.all(loadPromises).then(() => {
-      const allCall: Record<string, DcCall[]> = {};
+      const tmp: Record<number, Record<number, number>> = {};
       for (let i = 1; i < numRows; i++) {
         const row = i;
-        const queryId = String(sheet.getCell(i, header[QUERY_ID_COL]).value);
-        const callId = String(sheet.getCell(i, header[CALL_ID_COL]).value);
-        if (!allCall[queryId]) {
-          allCall[queryId] = [];
+        const queryId = Number(sheet.getCell(i, header[QUERY_ID_COL]).value);
+        const callId = Number(sheet.getCell(i, header[CALL_ID_COL]).value);
+        if (!tmp[queryId]) {
+          tmp[queryId] = {};
         }
-        allCall[queryId].push({ id: callId, row });
+        tmp[queryId][callId] = row;
       }
-      setAllCall(allCall);
+      setAllCall(tmp);
```

Comment:
nit: why use `tmp` as the name for the variable even though it's not really temporary because the object is the final object that gets set?

Reply:
using allCall would have a name collision with the state name, which i feel could be confusing.